### PR TITLE
 finish_prum helper

### PR DESCRIPTION
**Pull request by dy2403**
Hi, prof. @sbillinge,

I added the tests to the finish_prum, can you please take a look?

### CODE REVIEW

File: tests/test_helpers.py [hunk 1]
```diff
@@ -115,6 +115,16 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    (["helper", "finish_prum", "--list"],
+     "1. 20ly_newprojectum      status: started      lead: lyang\n"
+     "2. 20sb_firstprojectum      status: proposed      lead: ascopatz\n"
+     ),
+    (["helper", "finish_prum", "--index", "1"],
+     "20ly_newprojectum status have been updated to finished.\n"
```
Comment (sbillinge) [inline tests/test_helpers.py:123]: have -> has

Comment (dy2403) [reply tests/test_helpers.py]: Done!

**Comment by dy2403** [comment]
Hi @sbillinge!
I changed the tests, taking an approach similar to u_milestone, as I think it would make it more intuitive. Can you take a look at this?

File: tests/test_helpers.py [hunk 2]
```diff
@@ -115,6 +115,14 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    (["helper", "finish_prum", "20sb_firstprojectum", "--end_date", "2020-07-01"],
+     "20sb_firstprojectum status has been updated to finished\n"
+     ),
+    (["helper", "finish_prum", "20sb"],
+     "Projecta not found. Projecta with similar names: \n"
```
Comment (sbillinge) [inline tests/test_helpers.py:122]: "Projectum not found. Projecta with similar names: \n"

Comment (dy2403) [reply tests/test_helpers.py]: done!

**Comment by dy2403** [comment]
Hi @sbillinge!
Ready for review and merge!

**Comment by sbillinge** [comment]
great, thanks @dy2403 I merged.  Let's test this.  Since there is no group meeting this Friday, maybe we can roll it out at morning check-in on hte following Monday?  Please could you remind me?  We could roll it out before too, since I think it could be helpful to people!

**Comment by dy2403** [comment]
Sure! @sbillinge 
I think Monday would be good, so we have time to test it and correct possible bugs!

**Pull request merged**